Adyen/Core/Payment Methods/OnlineBankingPaymentMethod.swift

```diff
@@ -19,6 +19,16 @@ public struct Issuer: Codable, CustomStringConvertible, Equatable {
         name
     }
 
+    /// Initializes a new instance of `Issuer`.
+    ///
+    /// - Parameters:
+    ///   - identifier: The identifier.
+    ///   - name: The name.
+    public init(identifier: String, name: String) {
```

Comment:
Does this have to be public? Or can we use `@_spi`

Reply:
This was not needed.